### PR TITLE
KeepKey, Ledger Web Wallet and Electrum separation

### PR DESCRIPTION
**Pull request by maxtannahill**

### CODE REVIEW

**Comment by nvk** [comment]
Before we merge and to avoid confusion I think we should do that to all HW. I also think we should keep the HWs at the top when unpaired like we had.

ie HW vs HW+SW

Thoughts?

**Comment by maxtannahill** [comment]
Yes, good with that.

File: README.md [hunk 1]
```diff
@@ -20,9 +20,7 @@ If we went to your website and couldn't find it => ☠️☠️☠️ [EXTERNAL
 Status|Hardware Wallets|Supported Paths|BIP39 Pass|BIP174 PSBT|Note
 :---:|:---|:---:|---|---|---
 ✅|ColdCard[↗︎](https://coldcardwallet.com/)|`m/44'`\|`49'`\|`84'/0'/0'` + Custom|Optional|Yes|[Docs](https://coldcardwallet.com/docs/)
-✅|Trezor One / T[↗︎](https://trezor.com)|`m/44'`\|`49'/0'/0'`|Optional||[Docs](https://wiki.trezor.io)
```
Comment (nvk) [inline README.md]: Don't remove from the top.

File: README.md [hunk 2]
```diff
@@ -20,9 +20,7 @@ If we went to your website and couldn't find it => ☠️☠️☠️ [EXTERNAL
...
 ✅|Opendime[↗︎](https://opendime.com)|WIF|||[Docs](https://opendime.com/faq), [Archive](https://web.archive.org/save/https://opendime.com/faq)
-✅|KeepKey[↗︎](https://keepkey.zendesk.com/hc/en-us)|`m/44'/0'/0'`|Optional||[Docs](https://keepkey.zendesk.com/hc/en-us/articles/360001449050-How-to-Recover-on-your-KeepKey), [xPub](https://keepkey.zendesk.com/hc/en-us/sections/360000520290-Software), [Compatible Wallets](https://keepkey.zendesk.com/hc/en-us/articles/360001279444-What-Type-of-Wallet-Can-be-Used-to-Recover-KeepKey-)|
```
Comment (nvk) [inline README.md]: Don't remove from the top.

**Comment by nvk** [comment]
Apologies for all the rebasing, the pains of editing Markdown tables in group.

**Comment by nvk** [comment]
I've pulled this changes by hand. Not sure what you mean by your SMS number.

**Comment by maxtannahill** [comment]
This looks much better. Thanks.

**Pull request closed without merging**